ledger_device_sdk/src/testing.rs

```diff
@@ -53,28 +53,26 @@ pub struct TestType {
 /// using semihosting. Only reports 'Ok' or 'fail'.
 #[cfg(feature = "speculos")]
 pub fn sdk_test_runner(tests: &[&TestType]) {
-    use core::ffi::c_void;
-    use ledger_secure_sdk_sys::{pic, pic_rs};
     let mut failures = 0;
     debug_print("--- Tests ---\n");
     for test_ in tests {
         // (ノಠ益ಠ)ノ彡ꓛIꓒ
-        let test = pic_rs(*test_);
+        let test = *test_;
```

Comment:
Why did this Pull Request remove the `PIC` macro? It feels like such a change should be split in a different PR and reviewed on its own (because I am not knowledgeable enough about the state of application loading and whether we still need `PIC` (and `pic_rs`) in embedded applications.

Reply:
Well, `PIC` macro are no more needed following #128, so I did a little cleanup in the test runner used on `speculos`. I agree that it could have been proposed as a different PR but the impact is quite limited as it only concerns tests on speculos.